rust/template/src/main.rs

```diff
@@ -95,7 +94,7 @@ fn handle_cmd(
         Command::Comment => Ok(()),
         Command::Rollback => hddlog.transaction_rollback(),
         Command::Timestamp => {
-            println!("Timestamp: {}", precise_time_ns());
+            println!("Timestamp: {:#?}", std::time::Instant::now());
```

Comment:
why did you have to change this. The new method will produce output that does not look good, e.g., `Instant { tv_sec: 5122222, tv_nsec: 893024142 }`